thoth/messaging/__init__.py

```diff
@@ -34,6 +34,17 @@
     UnresolvedPackageMessage,
 ]
 
+__all__ = [
```

Comment:
```
__all__ = ALL_MESSAGES + ["MessageBase", "message_factory"]
```

Reply:
`ALL_MESSAGES` contains class references and `__all__` contains strings

Reply:
sorry, my bad :/

Reply:
```python
[msg_cls.__name__ for msg_cls in ALL_MESSAGES]
```